k8s/configuration/config-map-tm.yaml

```diff
@@ -0,0 +1,177 @@
+## Note: data values do NOT have to be base64-encoded in this file.
+
+## vars is common environment variables for this BigchaindB node
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vars
+  namespace: default
+data:
+  # cluster-fqdn is the DNS name registered for your HTTPS certificate.
+  cluster-fqdn: "bdb.example.com"
+
+  # cluster-frontend-port is the port number on which this node's services
+  # are available to external clients.
+  cluster-frontend-port: "443"
+
+  # cluster-health-check-port is the port number on which an external load
+  # balancer can check the status/liveness of the external/public server.
+  # In our deployment, Kubernetes sends 'livenessProbes' to this port and
+  # interprets a successful response as a 'healthy' service.
+  cluster-health-check-port: "8888"
+
+  # cluster-dns-server-ip is the IP of the DNS server. A Kubernetes deployment
+  # always has a DNS server (kube-dns) running at 10.0.0.10
+  cluster-dns-server-ip: "10.0.0.10"
+
+  # mdb-instance-name is the name of the MongoDB instance in this cluster.
+  mdb-instance-name: "<name of the mdb instance>"
+
+  # ngx-instance-name is the name of the NGINX instance in this cluster.
+  ngx-instance-name: "<name of the nginx instance>"
+
+  # openresty-instance-name is the name of the OpenResty instance in this
+  # cluster.
+  openresty-instance-name: "<name of the openresty instance>"
+
+  # bdb-instance-name is the name of the BigchainDB instance in this cluster.
+  bdb-instance-name: "<name of the bdb instance>"
+
+  # mdb-mon-instance-name is the name of the MongoDB Monitoring Agent instance
+  # in this cluster.
+  mdb-mon-instance-name: "<name of the mdb monitoring agent instance>"
+
+  # mdb-bak-instance-name is the name of the MongoDB Backup Agent instance
+  # in this cluster.
+  mdb-bak-instance-name: "<name of the mdb backup agent instance>"
```

Comment:
The MongoDB Backup instance name can be removed, no?

Reply:
yes. Done.

Reply:
:+1: